### PR TITLE
[MADPT1121] Calvin Turner DONE

### PR DESCRIPTION
**Pull request by Calvin087**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,86 @@
...
-          Learn React
-        </a>
-      </header>
+    <div
+      className="App"
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+        gap: '20px',
```
Comment (manucaralmo) [inline src/App.js:19]: It is better not to use inline styles, let's avoid it as much as possible. In a large project it is not maintainable to write the styles like this.

File: src/components/BoxColour/BoxColour.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+class BoxColour extends Component {
+  constructor(props) {
+    super(props);
+    this.myBackgroundColour = `rgb(${this.props.r},${this.props.g},${this.props.b})`;
+  }
+
+  displayHex() {
+    let a = this.myBackgroundColour.split('(')[1].split(')')[0];
+    let b = a.split(',').map((el) => {
+      let x = parseInt(el).toString(16);
```
Comment (manucaralmo) [inline src/components/BoxColour/BoxColour.js:13]: If the value is not going to change, it is better to always use `const`.

File: src/components/Rating/Rating.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    let ratings = [];
+
+    for (let i = 0; i < this.num; i++) {
+      ratings.push('★');
+    }
+
+    while (ratings.length < 5) {
+      ratings.push('☆');
+    }
+    return ratings;
```
Comment (manucaralmo) [inline src/components/Rating/Rating.js:19]: Maybe, a good solution could be to use `'★'.repeat(this.num) + '☆'.repeat(5 - this.num)`

**Review by manucaralmo — COMMENTED** [review]
Good work Calvin!

**Comment by Calvin087** [comment]
@manucaralmo 
I forgot .repeat() existed 🤦🏽‍♂️
Thank you.

**Pull request closed without merging**